src/base/preferences.cpp

```diff
@@ -804,6 +804,26 @@ void Preferences::setAutoRunProgram(const QString &program)
     setValue("AutoRun/program", program);
 }
 
+#if defined(Q_OS_WIN) && (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
+bool Preferences::isAutoRunConsoleEnabled() const
+{
+#if DISABLE_GUI
```

Comment:
Maybe it is just me, I felt weird for seeing `DISABLE_GUI` mentioned in Preferences class. If possible I would just ignore the `DISABLE_GUI` case.
Also IIRC on windows, qbt can also be built with `DISABLE_GUI` defined so maybe you will need to add `DISABLE_GUI` check to application.cpp

Reply:
I now realized that the nox build can be used in an GUI OS. So just basically load/save the value normally, and let the user manually edit the conf file.
Is that ok?

Reply:
seem yes.

Reply:
done